backend-lambda/index.js

```diff
@@ -42,21 +42,19 @@ export const handler = async (event, context) => {
       response = await productService.createProduct(pool, body);
     }
   }
-
-  // delete a product
-  else if (path.includes("/product/delete") && httpMethod === "DELETE") {
-    const id = await validate.validateProductId(
-      event.pathParameters.product_id
-    );
-    response = await productService.deleteProduct(pool, id);
-  }
-
   // update a product
-  else if (path.startsWith("/product/") && httpMethod === "PUT") {
-    const id = await validate.validateProductId(
-      event.pathParameters.product_id
-    );
-    response = await productService.updateProduct(pool, id, body);
+  else if (path.startsWith("/product/") && event.pathParameters.product_id) {
```

Comment:
need to remove the product id check from validate function

Reply:
if the product id isn't in the path will JS know to tell us that?

Reply:
You're right. That's redundant. Thanks!